src/eeg.jl

```diff
@@ -70,26 +82,40 @@ function labels2positions(labels)
         if haskey(CHANNEL_TO_POSITION_10_20, key)
             return CHANNEL_TO_POSITION_10_20[key]
         else
-            error("Currently only 10_20 is supported. Found: $(label)")
+            error("Currently only 10_20 is supported. Found label: $(label)")

```

Comment:
```suggestion
            error("Currently only 10/20 is supported. Found label: $(label)")

```

to make it consistent with other spots in the docs, even with have variables with `_`